hypermap/hypermap/urls.py

```diff
@@ -11,14 +12,14 @@
     url(r'^layer/(?P<layer_id>\d+)/map(?P<path_info>/.*)$', layer_mapproxy, name='layer_mapproxy'),
     url(r'^layer/(?P<layer_id>\d+)/tms/(?P<z>\d+)/(?P<y>\d+)/(?P<x>\d+).png$', layer_tms, name='layer_tms'),
     url(r'^layer/(?P<layer_id>\d+)/map/config$', layer_tms, name='layer_tms'),
+    url(r'^csw$', csw_global_dispatch, name='csw_global_dispatch'),
 
     (r'^', include('aggregator.urls')),
 )
 
-if settings.DEBUG:
-    urlpatterns += patterns(
-        '',
-        url(r'^media/(?P<path>.*)$',
-            'django.views.static.serve',
-            {'document_root': settings.MEDIA_ROOT, }),
-    )
+urlpatterns += patterns(
```

Comment:
We are deploying this in an environment that does not let us configure nginx.

In your case, nginx will serve this request before it gets to django, so it is  non issue.